src/EVM/Effects.hs

```diff
@@ -46,6 +46,7 @@ data Config = Config
   , maxBranch     :: Int
   , promiseNoReent   :: Bool
   , maxBufSize       :: Int
+  , verb        :: Int
```

Comment:
Should this be aligned with the other options? Also `maxBranch` seems like it is standing out.

Reply:
OOops, yes, let me fix.